File: tests/test-sample.R
# tests/test-sample.R
#
# This is a sample test file to demonstrate the structure of tests.
# We will use the 'testthat' library for testing our R code.

library(testthat)

test_that("Basic addition works", {
  expect_equal(1 + 1, 2)
})

File: tests/test_sample.py
# tests/test_sample.py
#
# This is a sample test file to demonstrate testing for Python code.
# We will use the 'pytest' framework.

# A simple function to test
def add(a, b):
    return a + b

# A test for the function
def test_add():
    assert add(1, 1) == 2
